includes/Shortcode/Embed_Shortcode.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Class Embed_Shortcode.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Shortcode;
+
+use Google\Web_Stories\Embed_Base;
+
+/**
+ * Embed shortcode class.
+ */
+class Embed_Shortcode extends Embed_Base {
+	/**
+	 * Shortcode name.
+	 *
+	 * @var string
+	 */
+	const SHORTCODE_NAME = 'web_stories_embed';
+
+	/**
+	 * Initializes the Web Stories embed shortcode.
+	 *
+	 * @since 1.1.0
+	 *
+	 * @return void
+	 */
+	public function init() {
+		add_shortcode( self::SHORTCODE_NAME, [ $this, 'render_shortcode' ] );
+	}
+
+	/**
+	 * Renders the shortcode  for given attributes.
+	 *
+	 * @since 1.1.0
+	 *
+	 * @param array  $attributes Shortcode attributes.
+	 * @param string $content Shortcode content.
+	 *
+	 * @return string Rendered Shortcode
+	 */
+	public function render_shortcode( array $attributes, $content ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Comment:
@spacedmonkey Should this callback use `shortcode_atts()`? IIRC it's recommended to use that, to allow filtering by plugins.

Reply:
Good catch. Fixed in https://github.com/google/web-stories-wp/pull/4780/commits/9f0193e27c7b528a0d40286d39d547d17316f4c2